.github/ISSUE_TEMPLATE/01-bug-report.yml

```diff
@@ -0,0 +1,51 @@
+name: Report a Technical/Visual Issue on the Node.js API Docs 
```

Comment:
react components are on the `node/nodejs.org` repo so do we need this template here or should we redirect to the `node/nodejs.org` repo?

Reply:
I thought we going to add some react components here for the docs?

Reply:
Yeah this is a tricky one. I think we should keep it as this repository. Because although the shared components are on the website repo, the layouts will be here. So it makes more sense for it to be reported here.

Although, Im pretty sure people will open issue on the other repository (nodejs.org) simply because the link for most people will be nodejs.org/api 🤷

So not sure what to do here. Fine either ways

Reply:
As far as I'm concerned, we can leave it like that and change it if need be.